src/utils/ApiClient.js

```diff
@@ -131,6 +135,9 @@ export default class ApiClient {
                 .put(`${this.server}/rest/v4/${partialUrl}`)
                 .auth(this.username, this.password)
                 .set("User-Agent", `Hyperwallet Node SDK v${this.version}`)
+                .set("x-sdk-version", this.version)
```

Comment:
since we are repeating the header parameters in all these methods, can we create another method that return `request` with headers already been defined?

```
prepareRequest() {
   return request.
                .auth(this.username, this.password)
                .set("User-Agent", `Hyperwallet Node SDK v${this.version}`)
                .set("x-sdk-version", this.version)
                .set("x-sdk-type", "NodeJS")
                .set("x-sdk-contextId", this.contextId);
}
```

Reply:
Externalizing the logic results in errors. But this can be done with little more time and some refactoring. This can be taken up for the next release. Please approve if this is okay.